core/pom.xml

```diff
@@ -47,6 +47,16 @@
   <build>
     <plugins>
 
+	
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-eclipse-plugin</artifactId>
+          <configuration>
+              <downloadSources>true</downloadSources>
+              <downloadJavadocs>true</downloadJavadocs>
+          </configuration>
+      </plugin>
```

Comment:
Is this plugin needed or is it just for debugging?

Reply:
Oh.. I forgot to remove that again. Although it definitely doesn't hurt to have it in there, it is not needed. Will remove again.